### DISCUSSION TITLE
Troubleshooting

### DISCUSSION THREAD

**Discussion by Cristalnoir** [Mezryss/FVTT-Genesys#150] [Ideas]
Create a discussion folder for errors encountered.
impossible to drag a career onto the Character Sheet

**Comment by Assembling-Kings** [comment]
> Impossible to drag a career onto the Character Sheet

This is not enough data to troubleshoot what you're experiencing.

**Comment by Cristalnoir** [comment]
I created a career with the appropriate menu. Then, I create a character by putting a name, an archetype, and when I drag the career created previously, it does not want to appear on the person sheet. The “Career” field remains empty.

**Reply by Assembling-Kings** [reply]
Disable modules and try again because this is working properly. If you're still experiencing problems then I'll need more info like Foundry version, system version, the contents of the Career, etc.

**Reply by Cristalnoir** [reply]
I uninstalled the module then reinstalled, it still doesn't work. 
Foundry VTT Version 11 stable build 315
Genesys alpha 0.2.9
I created these careers on V0.2.8. So I recreated new careers in 0.2.9. But impossible to assign them on the character sheet

**Reply by Cristalnoir** [reply]
bug fixed. The problem was not career or race. It came from the character sheet.
I recreated another character and everything works fine.
